aspnet/fundamentals/configuration.rst

```diff
@@ -41,7 +40,6 @@ It's not unusual to store configuration values in a hierarchical structure, espe
 .. _config-json:
 
 .. literalinclude:: /../common/samples/WebApplication1/src/WebApplication1/appsettings.json
-  :linenos:
   :language: json
 
 The application uses configuration to configure the right connection string. Access to the ``ConnectionString`` setting is achieved through this key: ``Data:DefaultConnection:ConnectionString``.
```

Comment:
One more bit of feedback about the custom EntityFramework configuration provider, its a bit confusing seeing builder.Build() twice, instead can we do something like the following?

Maybe call it AddDatabase() instead of AddEntityFramework so it doesn't sound like the AddEntityFramework service collection method.  And have the source directly take the connection string, which you can build a configuration instance who's sole purpose is to read that configuration for the source, rather than reusing the same instance of a ConfigurationBuilder which should make it easier to follow.